.gitignore

```diff
@@ -9,3 +9,10 @@ keras/datasets/temp/*
 docs/site/*
 docs/theme/*
 tags
+
+# test-related
+.coverage
+.cache
+
+# developer environments
+.idea
```

Comment:
.coverage
.cache
coverage files that you get in you working tree if you run tests locally

.idea is pycharm